packages/appstore/src/constants/platform-config.ts

```diff
@@ -82,3 +81,40 @@ export const DERIV_PLATFORM_NAMES = {
     BBOT: 'Binary Bot',
     GO: 'Deriv GO',
 } as const;
+
+export const MOBILE_PLATFORMS = {
+    IOS: 'ios',
+    HAUWEI: 'huawei',
+    ANDROID: 'android',
+} as const;
+
+type TMobilePlatforms = typeof MOBILE_PLATFORMS[keyof typeof MOBILE_PLATFORMS];
+
+const DERIVGO_IOS_APP_URL = 'https://apps.apple.com/my/app/deriv-go-online-trading-app/id1550561298';
```

Comment:
@sergei-deriv better to move these urls into this file : packages/appstore/src/constants/platform-config.ts

Reply:
@mahdiyeh-deriv excuse me, maybe you meant another file, but these links are in this file already :)

Reply:
My bad, it's the correct file